signer.go

```diff
@@ -24,12 +24,12 @@ func (s *Signer) Sign(
 	ctx context.Context,
 	relayRequest *servicetypes.RelayRequest,
 	// TODO_IMPROVE: this input argument should be changed to an interface.
-	app ApplicationRing,
+	appRing ApplicationRing,
```

Comment:
Do we need the `queryHeight` input argument after this change?

Reply:
Good catch!